CryptoLightNews/Model/NewsArticles.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NewsArticles.swift
+//  CryptoLightNews
+//
+//  Created by Richard Price on 21/12/2022.
+//
+
+import Foundation
+
+struct NewsArticles: Codable {
+    var title: String?
+    var url: String?
+    var urlToImage: String?
+    var author: String?
+    
+    enum CodingKeys: String, CodingKey {
```

Comment:
Do you need to create the CodingKeys here? It the keys are the same as your properties' names, you don't need them.